exercises/Makefile

```diff
@@ -0,0 +1,140 @@
+# Colors
+COLOR=\033[0;32m
+NC=\033[0m
+BOLD=\033[1m
+
+# Vars
+DOCKER_COMPOSE_CMD  = docker-compose -f docker-compose.yml
+KUBECTL_CONFIG_PATH = .output/kubeconfig.yaml
+KUBECTL_CMD         = kubectl --kubeconfig=${KUBECTL_CONFIG_PATH}
+
+##
+## Targets to manage the local k3s cluster
+## ---------------------------------------
+##
+
+all: help
+help: ## Display this message
+	@grep -E '(^[a-zA-Z0-9_.-]+:.*?##.*$$)|(^##)' Makefile | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
+
+start: ## Start k3s cluster
```

Comment:
Do we want to do a stop before a start ? Is there any case when we'd like to clean an old cluster ?

Reply:
I'd say no, since most process control systems (systemd, docker-compose, etc…) don't do that on a `start` action.

Reply:
because of this: https://github.com/hugoShaka/k8s-workshop/blob/master/exercises/docker-compose.yml#L20

Reply:
Fixed. 
Sorry, missed to remove it after a test session.